stdlib/enum.pyi

```diff
@@ -237,6 +237,8 @@ if sys.version_info >= (3, 11):
         _value_: str
         @_magic_enum_attr
         def value(self) -> str: ...
+        @staticmethod
+        def _generate_next_value_(name: str, start: Unused, count: Unused, last_values: Unused) -> str: ...
```

Comment:
The idea of `_generate_next_value_` is that you can override it in subclasses of `Enum` to provide custom behaviour for how `auto()` works. As such, I don't think we should use `object` annotations (which is what `Unused` is) for the parameters, or subclassers won't get the correct signature if they do actually want to use the parameters.

```suggestion
        def _generate_next_value_(name: str, start: int, count: int, last_values: list[str]) -> str: ...
```